openshift/obsidian-generator-backend.app.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: generator-backend
+metadata:
+  name: generator-backend
+  creationTimestamp: null
+objects:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    creationTimestamp: null
+    labels:
+      service: generator-backend
+    name: generator-backend
+  spec:
+    ports:
+    - name: "8080"
+      port: 8080
+      protocol: TCP
+      targetPort: 8080
+    selector:
+      service: generator-backend
+    sessionAffinity: None
+    type: ClusterIP
+  status:
+    loadBalancer: {}
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    creationTimestamp: null
+    name: generator-backend
+    generation: 1
+  spec:
+    replicas: 1
+    selector:
+      service: generator-backend
+    strategy:
+      resources: {}
+      rollingParams:
+        intervalSeconds: 1
+        maxSurge: 25%
+        maxUnavailable: 25%
+        timeoutSeconds: 600
+        updatePeriodSeconds: 1
+      type: Rolling
+    template:
+      metadata:
+        creationTimestamp: null
+        labels:
+          service: generator-backend
+      spec:
+        containers:
+        - image: registry.devshift.net/obsidian/obsidian-generator-backend:latest
```

Comment:
Perhaps it would be better if we could point to the Docker Hub image instead: https://hub.docker.com/r/redhatdevelopers/